### ISSUE TITLE
Difficulty to use VIM on Windows 7 client

### ISSUE THREAD

**Issue by PorkyProgs** [VSCodeVim/Vim#176]
Hi,
 I'm not really sure where I should ask this question, but this seams to be the best place since it concerns the VIM extension,
I installed the VIM extension on my Windows 7 SP1, .Net 4.5 Client with Visual Studio Code version 0.10.8 and when I try to edit a file I constantly receive the following warning for each key that I press:

> No handler found for the command: 'extension.vim_s'. Ensure there is an activation event defined, if you are an extension.

I checked in my profile .vscode\extensions\vscodevim.vim-0.0.11 folder package.json file which contains:

>  "activationEvents": [
    "*"
  ],

Perhaps someone can tell me where I am going wrong or what I should be doing to use the VIM editing functions. Pressing ':' or Ctrl+[ does not seem to work. I am using the international en-US keyboard.
Thanks, 
Porky

**Comment by jpoon** [comment]
Have you tried uninstalling and reinstalling? :smile: 

I am totally *serious*. I've noticed that sometimes the installation is a little finicky. You should see something like this in your extensions folder (`.vscode\extensions\vscodevim.vim-0.0.11`):

<img width="164" alt="extensions_folder" src="https://cloud.githubusercontent.com/assets/85374/13535796/1f1519e8-e1f1-11e5-8660-b3462f77bc04.png">

**Comment by PorkyProgs** [comment]
hi jpoon, That was an interesting suggestion, I had already done that however it did not resolve the problem, then when I looked at the problem close I discovered that I was missing the package.json file. I then created this file by making a copy from the github version  online. 
Perhaps I should uninstall, reboot and then reinstall again? Nevertheless, it's going to have to wait untill Monday as the machine is in the office.
